pages/caritonaranjo/index.html

```diff
@@ -0,0 +1,93 @@
+    <section id="carousel">
+      <h1 class="carousel__title">
+        OUR <span>BOOKS</span>
+      </h1>
+
+            <!--Slides-->
+      <div class="carousel__block">
+        <div class="card__block">
+          <div class="card__image">
+            <img class="card__img--top" src="images/history.jpg" alt="Card image" />
+          </div>
+            <div class="card__body">
+              <h4 class="card__title">HISTORY</h4>
+              <p class="card__text">
+                History (from Greek ἱστορία - historia, meaning "inquiry, knowledge acquired by investigation") is the discovery, collection, organization, and presentation of information about past events. 
+                History can also mean the period of time after writing was invented. Scholars who write about history are called historians.
+              </p>
+              <button class="button__primary">MORE</button>
+            </div>
+        </div>
+
+        <div class="card__block">
+          <div class="card__image">
+            <img class="card__img--top" src="images/romance.jpeg" alt="Card image" />
+          </div>
+            <div class="card__body">
+              <h4 class="card__title">ROMANCE</h4>
+              <p class="card__text">
+                According to the Romance Writers of America, "Two basic elements comprise every romance novel: a central love story and an emotionally-satisfying and optimistic ending.
+                " Both the conflict and the climax of the novel should be directly related to that core theme of developing a romantic relationship, although the novel can also contain subplots that do not specifically relate to the main characters' romantic love.
+              </p>
+              <button class="button__primary">MORE</button>
+            </div>
+        </div>
+
+        <div class="card__block">
+          <div class="card__image">
+            <img class="card__img--top" src="images/fantasy.jpg" alt="Card image" />
+          </div>
+            <div class="card__body">
+              <h4 class="card__title">FANTASY</h4>
+              <p class="card__text">
+                Fantasy is a genre that uses magic and other supernatural forms as a primary element of plot, theme, and/or setting.
+                Fantasy is generally distinguished from science fiction and horror by the expectation that it steers clear of technological and macabre themes, respectively, though there is a great deal of overlap between the three (collectively known as speculative fiction or science fiction/fantasy)
+
+              </p>
+              <button class="button__primary">MORE</button>
+            </div>
+        </div>
+        <!--Controls-->
+        <section class="controls__side">
+          <button class="btn__side--right" href="#carousel" data-slide="prev"> 
+          <span class="btn__img "><</span>
+        </button>
+          <button class=" btn__side--left" href="#carousel" data-slide="next">
```

Comment:
Please remove the blank space after the double quotes and before the class btn__side--left